lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -10,7 +10,6 @@ module RSpec
       # include `if`, `unless`, `for`, `before`, `after`, or `during`.
       # They may consist of multiple words if desired.
       #
-      # @see https://rspec.rubystyle.guide/#context-descriptions
```

Comment:
[memo] The link to StyleGuide has been removed since it is already included in the Reference.